### ISSUE TITLE
Saveing Acex fortify budget   Request/Enhancement

### ISSUE THREAD

**Issue by tandkjaer** [gruppe-adler/grad-persistence#37]
Would i be possible to add Acex Fortify budget to the variables that are saved ?

**Comment by McDiod** [comment]
Yes, you need to find the relevant ACEX variables [here](https://github.com/acemod/ACEX/tree/master/addons/fortify) and then follow the instructions [here](https://github.com/gruppe-adler/grad-persistence/wiki/saving-variables).

I'm not familiar with ACEX fortify, but having a quick look at the code there seems to be only one variable per side. Your `cfgGradPersistence` should look something like this:

```sqf
class CfgGradPersistence {
    class customVariables {
        class acex_fortify_budget_west {
            varName = "acex_fortify_budget_west";
            varNamespace = "mission";
            public = 1;
        };
        class acex_fortify_budget_east: acex_fortify_budget_west {
            varName = "acex_fortify_budget_east";
        };
        class acex_fortify_budget_independent: acex_fortify_budget_west {
            varName = "acex_fortify_budget_independent";
        };
        class acex_fortify_budget_civilian: acex_fortify_budget_west {
            varName = "acex_fortify_budget_civilian";
        };
    };
};
```

You might also have to wait with loading this, otherwise ACEX's initialization might happen after the loading is done and will overwrite your loaded values. See [missionWaitCondition](https://github.com/gruppe-adler/grad-persistence/wiki/Configuration).

**Comment by tandkjaer** [comment]
class CfgGradPersistence {
    missionTag = "RDF_Torsdags_Mission";
    loadOnMissionStart = 1;
    missionWaitCondition = "true";
    playerWaitCondition = "true";
    
    saveUnits = 3;
    saveVehicles = 3;
    saveContainers = 3;
    saveStatics = 3;
    saveMarkers = 0;
    saveTasks = 0;
    saveTriggers = 0;

    savePlayerInventory = 1;
    savePlayerDamage = 1;
    savePlayerPosition = 1;

    class customVariables {
         class acex_fortify_budget_west {
            varName = "acex_fortify_budget_west";
            varNamespace = "mission";
            public = 1;
         };  
        
    };
};

Tried this to no avail.. have also tried setting the acex_forti... to something then saving then changing acex_fortify... and the using #gradpersistenceload and that dosent do anything either.

**Comment by McDiod** [comment]
Possibly fixed in [0.10.4](https://github.com/gruppe-adler/grad-persistence/releases/tag/0.10.4).

**Comment by tandkjaer** [comment]
It now savs and loads. but if i use the loadOnMissionStart the acex fortify modules sets the budget.. any way for delaying the load further ?

**Comment by McDiod** [comment]
Something like

```
missionWaitCondition = "!isNil 'acex_fortify_budget_west' || (missionNamespace getVariable ['CBA_missionTime',0]) > 60";
```

which will wait until either ACEX has set the budget variable or one minute has passed.

**Comment by tandkjaer** [comment]
The missionWaitCondition doset work.. but if i load the mission when on the server it works.

just to be sure the missionWaitConditon goes it the description.ext right ?`:)

**Comment by McDiod** [comment]
Yes, into your description.ext like so

```
class CfgGradPersistence {
    missionWaitCondition = "...";
};
```
